database/insertData/dev/04_users.js

```diff
@@ -93,6 +94,7 @@ exports.queries = [
           passwordHash: "$2a$10$ne2WcPISMw/Do3JzlwThYeO2GcodrumjI3FwGu1ZUoKgRQyAgNS3e"
           username: "valerio"
           firstName: "Valerio"
+          lastName: "R"
```

Comment:
This would still work if lastName wasn't specified, wouldn't it?

(I think it should)

Perhaps omit lastName from a couple of the users so we can see what happens/how it appears?

Reply:
Oops i never replied, I think the issue was with extra space at the end of applicant/assigners/reviewers column if last name is missing, which is not a biggy i guess